docs/content/stable/yugabyte-voyager/migrate/live-fall-forward.md

```diff
@@ -402,7 +417,12 @@ You can use only one of the following arguments in the `source` parameter (confi
 
 1. Grant permissions for migration. Use the [yb-voyager-pg-grant-migration-permissions.sql](../../reference/yb-voyager-pg-grant-migration-permissions/) script (in `/opt/yb-voyager/guardrails-scripts/` or, for brew, check in `$(brew --cellar)/yb-voyager@<voyagerversion>/<voyagerversion>`) to grant the required permissions as follows:
 
-    _Warning_: This script transfers ownership of all tables in the specified schemas to the specified replication group. The migration user and the original owner of the tables will be added to the replication group.
+    _Warning_: This script provides two options for granting permissions:
```

Comment:
Should this change be noted in RN? @makalaaneesh

Reply:
no, this change was made a while back. we just missed updating the documentation. 
https://deploy-preview-29802--infallible-bardeen-164bc9.netlify.app/stable/yugabyte-voyager/release-notes/#:~:text=Enhanced%20PostgreSQL%20permissions%20grant%20script%20by%20adding%20an%20option%20for%20live%20migrations%20to%20either%20transfer%20table%20ownership%20to%20the%20migration%20user%20or%20grant%20the%20original%20owner%27s%20permissions%20to%20it.